README.md

```diff
@@ -322,33 +322,32 @@ projects/
                                          config2.json
 ```
 
-#### Supported Configuration Types
+#### Supported Configuration Types and Token Permissions
 
 Supported configurations types are:
```

Comment:
Could we extend this intro line to make it clear that these are the configurations and the permissions that your env-token (making sure to reuse the wording we use in the rest of the documentation - not 100% this is it) will need to have